doc/rbd/rbd-encryption.rst

```diff
@@ -240,6 +240,17 @@ The same applies to creating a formatted clone of an unformatted
 (plaintext) image since an unformatted image does not have a header at
 all.
 
+To map a formatted clone of an encrypted parent image, provide the
+passphrase file of the clone and its encrypted parent image. The order in
+which the passphrase files should be provided is based on the image
+hierarchy, starting with that of the clone image, then its parent and so on.
```

Comment:
More precise language:
```suggestion
To map a formatted clone, provide encryption formats and passphrases
for the clone itself and all of its explicitly formatted parent images.
The order in which `encryption-format` and `encryption-passphrase-file`
options should be provided is based by the image hierarchy: start with
that of the cloned image, then its parent and so on.
```

Reply:
This has been committed as it stands here with the exception of "s/is based by/is based on/".

Reply:
I actually edited that before your comment ;)